### PR TITLE
Update export flow to match mocks

### PR DESCRIPTION
**Pull request by tjennison-google**

### CODE REVIEW

File: ui/src/export.tsx [hunk 1]
```diff
@@ -720,178 +730,315 @@ function ExportDialog(
...
+              if (exportState.data?.redirectURL) {
+                redirect(
+                  exportState.data.redirectURL,
+                  absoluteExportURL(params)
+                );
+              }
+            }}
+          >
+            Continue
+          </Button>
```
Comment (marikomedlock) [inline ui/src/export.tsx:986]: Does this bit give the user a option to Continue even if there were errors?

File: underlay/src/main/resources/config/underlay/sd/ui.json [hunk 2]
```diff
@@ -1,7 +1,4 @@
 {
-  "featureConfig": {
-    "disableExportButton": true
-  },
```
Comment (marikomedlock) [inline underlay/src/main/resources/config/underlay/sd/ui.json:4]: 🎉

**Review by marikomedlock — APPROVED** [review]

File: ui/src/export.tsx (hunk 1 above)
Comment (tjennison-google) [reply ui/src/export.tsx:986]: Yes, that's what was in the mocks. Do you think it shouldn't?

Comment (marikomedlock) [reply ui/src/export.tsx:986]: I think it's fine, as long as we also show the errors. Was just making sure I was following.

**Pull request merged**